CHANGES.rst

```diff
@@ -1,3 +1,517 @@
+  isophotes. [#15545]
+
+astropy.nddata
+^^^^^^^^^^^^^^
+
+- A more flexible and/or compact string representation is now available for
+  ``NDData`` objects which visually indicates masked entries, and provides for
+  better for dask array support. [#14438]
+
+astropy.table
+^^^^^^^^^^^^^
+
+- The new ``Row.get()`` method, analogous to ``dict.get()``, returns the value of
+  the specified column from the row if the column present, otherwise it returns a
+  fallback value, which by default is ``None``. [#14878]
+
+astropy.time
+^^^^^^^^^^^^
+
+- Masked ``Time`` instances now use astropy's own ``Masked`` class internally.
+  This means that ``Masked`` input is now properly recognized, and that masks
+  get propagated also to ``Quantity`` output (such as from a ``TimeDelta``
+  converted to a unit of time), creating ``MaskedQuantity`` instances. [#15231]
+
+- Added a ``TimeDelta`` format ``quantity_str`` that represents the time delta as a string
+  with one or more ``Quantity`` components. This format provides a human-readable
+  multi-scale string representation of a time delta. The default output sub-format is not
+  considered stable in this release, please see https://github.com/astropy/astropy/issues/15485
+  for more information. [#15264]
+
+astropy.uncertainty
+^^^^^^^^^^^^^^^^^^^
+
+- Uncertainty ``Distribution`` now support structured data types, and as
+  a result it now works also with ``EarthLocation``. [#15304]
+
+- Uncertainty ``Distribution`` can now be used inside representations, which
+  also allows basic support in ``SkyCoord``. While most calculations work, there
+  are remaining issues.  For instance, the ``repr`` does not show that the
+  coordinates are distributions. [#15395]
+
+astropy.units
+^^^^^^^^^^^^^
+
+- Add support for gc2gde and gd2gce erfa functions to allow geodetic representations
+  using equatorial radius and flattening. [#14729]
+
+astropy.utils
+^^^^^^^^^^^^^
+
+- The ``astropy.utils.metadata.MetaData`` default dictionary can now be
+  set with the ``default_factory`` keyword argument. [#15265]
+
+- ``astropy.utils.decorators.deprecated`` now adds the ``__deprecated__`` attribute to
+  the objects it wraps, following the practice in https://peps.python.org/pep-0702/. [#15310]
+
+astropy.visualization
+^^^^^^^^^^^^^^^^^^^^^
+
+- Add ``WCSAxes.text_coord`` method to print text using ``SkyCoord`` objects
+  parallel to plotting data points with ``WCSAxes.plot_coord``. [#14661]
+
+astropy.wcs
+^^^^^^^^^^^
+
+- Support WCS descriptions of basic planetary coordinate frames. [#14820]
+
+- Updated bundled WCSLIB version to 8.1. This update adds support planetary keywords ``A_RADIUS``, ``B_RADIUS``, ``C_RADIUS``, ``BLON_OBS``, ``BLAT_OBS``, and ``BDIS_OBS`` in ``auxprm`` and adds ``wcsprm::time`` to the ``wcsprm`` struct to record the ``TIME`` axis. This update also includes several bug fixes. For a full list of changes - see http://www.atnf.csiro.au/people/mcalabre/WCS/CHANGES [#15035]
```

Comment:
Not sure if any editing is wanted, but perhaps good to split this very long line (and, yes, I should have caught that in review...)